test/suites/edgexfoundry/main_test.go

```diff
@@ -52,23 +52,35 @@ func TestMain(m *testing.M) {
 }
 
 func TestCommon(t *testing.T) {
-	params := &utils.TestParams{
+	utils.WaitServiceOnline(t, 60, supportSchedulerServicePort)
+
+	// check network interface status for all platform ports except for:
+	// Kong’s port: 8000
+	// Kong-db's port: 5432
+	// Redis's port: 6379
+	var platformPorts []string
+	for _, port := range utils.PlatformPorts {
+		if port != "8000" && port != "5432" && port != "6379" {
+			platformPorts = append(platformPorts, port)
+		}
+	}
+
+	utils.TestCommon(t, utils.Params{
 		Snap: platformSnap,
 		App:  supportSchedulerApp,
-		TestConfigs: utils.TestConfigs{
+		ConfigTests: utils.ConfigTests{
 			TestEnvConfig:      utils.FullConfigTest,
 			TestAppConfig:      true,
 			TestGlobalConfig:   true,
 			TestMixedConfig:    utils.FullConfigTest,
-			DefaultServicePort: []string{supportSchedulerServicePort},
+			DefaultServicePort: supportSchedulerServicePort,
 		},
-		TestNetworking: utils.TestNetworking{
-			TestOpenPorts:        utils.PlatformPorts,
+		NetworkingTests: utils.NetworkingTests{
+			TestOpenPorts:        platformPorts,
```

Comment:
WaitServiceOnline(platformPorts) didn't test 8000, 5432 and 6379.
It needs utils.PlatformPorts here.